src/c_api/c_api.cc

```diff
@@ -1899,6 +1901,36 @@ int MXNDArraySave(const char* fname,
   API_END();
 }
 
+int MXNDArraySave(const char* fname,
+                  uint32_t num_args,
+                  NDArrayHandle* args,
+                  const char** keys) {
+  API_BEGIN();
+
+  CHECK_NOTNULL(fname);
```

Comment:
add meaningful error message?

Reply:
Modifying the message isn't supported as the macro is equal to an identity function if the argument is not null. The macro already defines an error message for the case of null arguments.